scripts/python/phdf_diff.py

```diff
@@ -110,12 +110,43 @@ def compare(files, all=False, brief=True, quiet=False, one=False, tol=1.0e-12):
         """)
         return(3)
 
+    if check_meta_data:
+        #Check in the metadata in Info is identical, except use tol for comparing "Time"
+        f0_Info = { key:value for key,value in f0.Info.items() if key != "Time" and key != "BlocksPerPE" }
+        f1_Info = { key:value for key,value in f1.Info.items() if key != "Time" and key != "BlocksPerPE" }
+
+        errTime = np.abs(f0.Time- f1.Time)
+        if errTime > tol:
+            print("""
+            Time of outputs differ
+
+            Quitting...
+            """)
```

Comment:
Because it is floating point I would recommend actually printing the difference in the error message.